Packs/KeeperSecretsManager/Integrations/KeeperSecretsManager/KeeperSecretsManager.py

```diff
@@ -0,0 +1,649 @@
+    return None
+
+
+@metadata_collector.command(command_name="ksm-get-infofile",
+                            inputs_list=[InputArgument(name="file_uid",
+                                                       required=True,
+                                                       description="File UID to search for."),
+                                         InputArgument(name="record_uid",
+                                                       required=False,
+                                                       default="",
+                                                       description="Record UID to search for files."
+                                                       " Search all records if empty.")],
+                            description="Use this command to fetch the file attachment as an Info File.")
+def get_infofile_command(client: Client, args: Dict[str, Any], **kwargs) -> Optional[dict]:
+    """Get info file command - returns the file attachment as an Info File.
+
+    Args:
+        client (Client): KeeperSecretsManager client to use.
+        file_uid (str):  The file UID to search for. required.
+        record_uid (str):  The record UID to search for files. Searches all records if record UID is empty.
+
+    :return:
+        A ``CommandResults`` object that is then passed to ``return_results``,
+        that contains the list with records and files info.
+
+    :rtype: ``CommandResults``
+    """
+
+    demisto.debug('get_infofile_command - command_name: ' + str(kwargs.get('command_name', '')))
+    record_uid = args.get('record_uid') or ''
+    file_uid = args.get('file_uid', None)
+    if not file_uid:
+        raise ValueError('file_uid not specified')
+    demisto.debug(f'Get file UID: "{file_uid}" from record UID:"{record_uid}"')
+
+    files = client.ksm_get_file(record_uid, file_uid)
+    if files:
+        file_result = fileResult(filename=files[0][0], data=files[0][1], file_type=EntryType.ENTRY_INFO_FILE)
+        # file_result["Contents"] = files[0][1]
+        return file_result
+    return None
+
+
+def test_module(client: Client) -> str:
+    """Tests API connectivity and authentication'
+
+    Returning 'ok' indicates that the integration works like it is supposed to.
+    Connection to the service is successful.
+    Raises exceptions if something goes wrong.
+
+    :type client: ``Client``
+    :param Client: client to use
+
+    :return: 'ok' if test passed, anything else will fail the test.
+    :rtype: ``str``
+    """
+
+    message: str = ''
+    try:
+        # Tests connectivity and authentication to KSM service.
+        # This validates all the inputs given in the integration configuration panel,
+        # either manually or by using an API that uses them.
+        client.secrets_manager.get_secrets(["AAAAAAAAAAAAAAAAAAAAA"])
+        message = 'ok'
+    except Exception as e:
+        message = str(e)
+    return message
+
+
+''' MAIN FUNCTION '''
+
+
+def main() -> None:
```

Comment:
Please add to increase coverage and [overcome build failure](https://app.circleci.com/pipelines/github/demisto/content/191887/workflows/2a73ea46-3107-4b7f-8f54-231d68186492/jobs/570866/steps?invite=true#step-108-14).

```suggestion
def main() -> None:   # pragma: no cover
```